### PR TITLE
Recommendation section

### PR DESCRIPTION
**Pull request by RalucaRbc**

### CODE REVIEW

**Review by Constantin-Matei — APPROVED** [review]
Inspirational for me :))

File: src/_component/portfolio/portfolio.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  margin-bottom: 0;
+}
+
+.old-projects {
+  margin-bottom: 10px;
+
+  &:last-of-type{
+    margin-bottom: 20px;
+  }
+}
```
Comment (blowymind99) [inline src/_component/portfolio/portfolio.scss:30]: from where is this one ?

**Review by blowymind99 — APPROVED** [review]
**Like it !**

**Review by marianzburlea — APPROVED** [review]

**Review by BogdanNegura — APPROVED** [review]
💯

**Review by BogdanNegura — APPROVED** [review]

**Pull request merged**